theano/gpuarray/opt.py

```diff
@@ -250,53 +260,69 @@ def apply(self, fgraph):
         # Iterating through inputs of graph
         for i in fgraph.inputs:
             if isinstance(i.type, tensor.TensorType):
```

Comment:
I guess this check isn't necessary anymore as the `as_gpuarray_variable` makes the check before returning `GpuFromHost` instance ? Also, I didn't see this wrapper over `GpuFromHost`.